src/main/kotlin/com/github/mitchellolsthoorn/testgenie/toolwindow/TestGenieToolWindow.kt

```diff
@@ -54,32 +57,72 @@ class TestGenieToolWindow {
      * Creates the entire tool window panel.
      */
     private fun createToolWindowPanel() = FormBuilder.createFormBuilder()
-            .setFormLeftIndent(30)
-            .addVerticalGap(25)
-            .addComponent(panelTitle)
-            .addLabeledComponent(customLabel("Search budget", "Maximum search duration."), searchBudget, 25, false)
-            .addTooltip(default("60 seconds"))
-            .addLabeledComponent(customLabel("Local search budget type", "Interpretation of local search budget value."), localSearchBudgetType, 20, false)
-            .addTooltip(default("Time"))
-            .addLabeledComponent(customLabel("Local search budget value", "Maximum budget usable for improving individuals per local search."), localSearchBudgetValue, 20, false)
-            .addTooltip(default("5"))
-            .addLabeledComponent(customLabel("Stopping condition", "What condition should be checked to end the search."), stoppingCondition, 20, false)
-            .addTooltip(default("60 seconds"))
-            .addLabeledComponent(customLabel("Initialization timeout", "Seconds allowed for initializing the search."), initializationTimeout, 20, false)
-            .addTooltip(default("120 seconds"))
-            .addLabeledComponent(customLabel("Minimisation timeout", "Seconds allowed for minimization at the end."), minimisationTimeout, 20, false)
-            .addTooltip(default("60 seconds"))
-            .addLabeledComponent(customLabel("Assertion timeout", "Seconds allowed for assertion generation at the end."), assertionTimeout, 20, false)
-            .addTooltip(default("60 seconds"))
-            .addLabeledComponent(customLabel("JUnit check timeout", "Seconds allowed for checking the generated JUnit files <p/>(e.g., compilation and stability)."), junitCheckTimeout, 20, false)
-            .addTooltip(default("60 seconds"))
-            .addLabeledComponent(customLabel("Population", "Population size of genetic algorithm."), population, 20, false)
-            .addTooltip(default("50"))
-            .addLabeledComponent(customLabel("Population limit", "What to use as limit for the population size."), populationLimit, 20, false)
-            .addTooltip(default("Individuals"))
-            .addComponent(createSaveAndResetButtons(), 20)
-            .addComponentFillVertically(JPanel(), 20)
-            .panel
+        .setFormLeftIndent(30)
+        .addVerticalGap(25)
+        .addComponent(panelTitle)
+        .addLabeledComponent(customLabel("Search budget", "Maximum search duration."), searchBudget, 25, false)
+        .addTooltip(default("60 seconds"))
+        .addLabeledComponent(
+            customLabel("Local search budget type", "Interpretation of local search budget value."),
+            localSearchBudgetType,
+            20,
+            false
+        )
```

Comment:
Why did you refactor the code that is not part of this merge request?

Reply:
I think the changes here come as a result of me merging #26 prematurely to my branch.